src/resolvers/userResolver.js

```diff
@@ -231,6 +231,58 @@ const userResolver = {
       );
     },
 
+    changePassword: async(_, { oldPassword, newPassword }, { req }) => {
+      const { password } = await User.findById(req.session.userId);
+      const isOldPassword = await bcrypt.compare(oldPassword, password);
```

Comment:
The variable `isOldPassword` can be more descriptive so it is easier to read. You might write it as `isOldPasswordCorrect` or something like it.